aten/src/ATen/native/cpu/UpSampleKernelAVXAntialias.h

```diff
@@ -409,7 +411,7 @@ void upsample_avx_bilinear_uint8(
 
     TORCH_INTERNAL_ASSERT(unpacked_output.defined());
 
-    if (!needs_unpacking) {
+    if (!skip_unpacking) {
```

Comment:
Here, should we define

```C
bool skip_repacking = (num_channels == 3 || num_channels == 4) && \
      outut.is_contiguous(at::MemoryFormat::ChannelsLast);
```

Jointly with the other suggestion above, this would avoid unpacking the input when the input is already in the right format?